aiida_quantumespresso/parsers/pp.py

```diff
@@ -145,8 +158,9 @@ def get_key_from_filename(filename):
         # If we don't have any parsed files, we exit. Note that this will not catch the case where there should be more
         # than one file, but the engine did not retrieve all of them. Since often we anyway don't know how many files
         # should be retrieved there really is no way to check this explicitly.
-        if not data_parsed:
-            return self.exit_codes.ERROR_OUTPUT_DATAFILE_MISSING.format(filename=filename_prefix)
+        #if not data_parsed:
+        #    return self.exit_codes.ERROR_OUTPUT_DATAFILE_MISSING.format(filename=filename_prefix)
+        # It is now also possible that nothing will be parsed
```

Comment:
If doing just step1 then may not retrieve any output. I could do more elaborate checks such as only checking this if the `PLOT` namelist was provided, etc. But since it anyways says that generically detecting missing files is too complicated, I'm not sure if this is even worth it.